maxim/maxim.py

```
@@ -945,15 +948,19 @@ def check():
             try:
                 exists = self.maxim_api.does_log_repository_exist(logger.id)
                 if not exists:
-                    scribe().warning(f"[MaximSDK] Log repository not found: {logger.id}. We will be dropping all logs.")
+                    scribe().warning(
+                        f"[MaximSDK] Log repository not found: {logger.id}. We will be dropping all logs."
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)